cardano-node/src/Cardano/Tracing/OrphanInstances/Shelley.hs

```diff
@@ -220,18 +220,20 @@ instance ToObject (Conway.ConwayDelegPredFailure era) where
       , "amount" .= coin
       , "error" .= String "Stake key has non-zero account balance"
       ]
-    Conway.DRepAlreadyRegisteredForStakeKeyDELEG credential ->
-      [ "kind" .= String "DRepAlreadyRegisteredForStakeKeyDELEG"
-      , "amount" .= String (textShow credential)
-      , "error" .= String "DRep already registered for the stake key"
-      ]
 
 instance ToObject (Set (Credential 'Staking StandardCrypto)) where
   toObject _verb creds =
     mconcat [ "kind" .= String "StakeCreds"
              , "stakeCreds" .= map toJSON (Set.toList creds)
              ]
 
+instance ToObject (NonEmpty.NonEmpty (KeyHash 'Staking StandardCrypto)) where
+  -- TODO fill in
+  toObject _verb _what =
+    mconcat [ "kind" .= String "????"
+            -- , "????" .= toObject "what"
+             ]
```

Comment:
Same thing for this one, just use the default instance otherwise implement it yourself